KinConnections/resources/email_wrapper.py

```diff
@@ -0,0 +1,96 @@
+# email_wrapper.py - KinConnections
+# Serves as wrapper for sending connection emails
+
+# for environment variables
+import os
+from dotenv import load_dotenv
+# for email engine
+import smtplib, ssl
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from email.utils import formataddr
+# for rich html email
+from dominate import document
+from dominate.tags import *
+# flask
+from flask import Blueprint, Response, request, render_template, session, redirect, url_for
+from flask_restful import Resource
+
+from .db_wrapper import *
+
+
+# access credentials from .env
+load_dotenv(dotenv_path="../.env")
+
+kc_email = os.getenv("KC_EMAIL")
+kc_email_password = os.getenv("KC_EMAIL_PASSWORD")
+kc_email_server = os.getenv("KC_EMAIL_SERVER")
+kc_email_port = int(os.getenv("KC_EMAIL_PORT"))
+
+class Email(Resource):
+    def get(self):
+        return 'Error! Return <a href="/">Home</a>'
+        
+    def post(self, id):
+        # ensure logged in
+        if not session.get('email', None):
+            return redirect("/login?error=notSignedIn")
+
+        # send email
+        subject = (request.form.get('userSubject'))
```

Comment:
Q: What are we putting these variables inside `()`. I am not familiar with that Python syntax so not sure what the purpose is here vs just doing a good ol' fashion assignment.